projects/js-toolkit/packages/liferay-cli/src/new/util/getPortletName.ts

```diff
@@ -0,0 +1,18 @@
+/**
+ * SPDX-FileCopyrightText: © 2017 Liferay, Inc. <https://liferay.com>
+ * SPDX-License-Identifier: LGPL-3.0-or-later
+ */
+
+/**
+ * Get the portlet name.
+ *
+ * @param pkgJsonName
+ */
+export default function getPortletName(pkgJsonName: string): string {
+	let portletName = pkgJsonName;
+
+	portletName = portletName.replace(/-/g, '');
+	portletName = portletName.replace(/[^A-Za-z0-9]/g, '_');
+
+	return portletName;
+}
```

Comment:
Is there any benefit to separating these into different lines?

```suggestion
export default function getPortletName(pkgJsonName: string): string {
  return pkgJsonName.replace(/-/g, "").replace(/[^A-Za-z0-9]/g, "_");
}

```

Reply:
Simply readability. But I'm OK with chaining too :shrug: